rocketpy/Flight.py

```diff
@@ -4466,31 +3744,31 @@ def allInfo(self):
         self.info()
```

Comment:
Shall we use `self.prints.all()` and `self.plots.all()` instead of invking each method separately?

Reply:
`Flight.allInfo()` doesn't show all the plots, I didn't use `self.plots.all()` for that reason

Reply:
> `Flight.allInfo()` doesn't show all the plots, I didn't use `self.plots.all()` for that reason

Mmmmm that's weird... why not? And don't worry, we don't need to get constrained by the current Flight.allInfo() behavior, the breaking change is already going to happen anyway.
I believe `self.plots.all()` would be more easy to understand since it applies the same idea of `self.prints.all()`, do you agree? Otherwise I don't see the point of having a all() method in the _FlightPlots class, makes sense?

Reply:
Agreed, I'll add self.plots.all() to allInfo